docs/doxygen/overviews/install.md

```diff
@@ -0,0 +1,106 @@
+  in system repositories under the name "wxGTK". Note that to develop
+  applications using wxWidgets you may need to install the "development"
+  packages and not just the libraries needed for running the applications using
+  wxWidgets. For example, under Debian and Debian-derived systems such as
+  Ubuntu, you need to run `apt get libwxgtkX.Y-dev`.
+
+- For macOS, wxWidgets is available in third-party package managers such as
+  [brew][1] or [MacPorts][2], and you can install them in the usual way.
+
+[1]: https://brew.sh/
+[2]: https://www.macports.org/
+
+
+Additionally, some third-party C++ package managers also provide wxWidgets
+binaries. For example, please see [this post][3] for the instructions about
+using [vcpkg][4] C++ package manager for installing wxWidgets.
+
+[3]: https://www.wxwidgets.org/blog/2019/01/wxwidgets-and-vcpkg/
+[4]: https://github.com/microsoft/vcpkg
+
+
+Building from Source
+--------------------
+
+### Getting the sources
+
+To build the library you need to get its sources first. The recommended way to
+do it is to use Git to check them out from the official wxWidgets repository
+using the following command:
+
+    $ git clone --recurse-submodules https://github.com/wxWidgets/wxWidgets.git
+
+Alternatively, you can download the sources from the [downloads page][1].
+Please note that all the source archives in different formats (ZIP, 7z,
+tar.bz2) contain the same files, but use different line ending formats: Unix
+("LF") for the latter one and DOS ("CR LF") for the two other ones, and it is
+usually preferable to choose the format corresponding to the current platform.
+When downloading the sources with DOS ends of lines, prefer 7z format for much
+smaller file size.
+
+
+### Selecting the build system
+
+wxWidgets can be built using [CMake](https://cmake.org/) under all platforms.
+Please follow [CMake build instructions](@ref overview_cmake) if you prefer to
+use it.
+
+Otherwise, please use the appropriate instructions depending on your platform:
+
+- For native wxMSW port under [Microsoft Windows](@ref plat_msw_install).
+- For wxGTK under [Linux and other Unix-like systems](@ref plat_gtk_install).
+- For native wxOSX port under [macOS](@ref plat_osx_install).
+
+The wxWidgets ports mentioned above are the main ones, however other variants
+also exist, see [platform details](@ref page_port) page for the full list.
+
+
+Verifying the Installation
+--------------------------
+
+After installing wxWidgets, it is recommended to check that the library can
+actually be used by building a minimal wxWidgets program using it, such as the
+"Hello world" example from the next section.
+
+To do it under Unix-like systems, including macOS and Unix-like environments
+such as MSYS2 under MSW, you can use the following command line:
+
+    $ c++ -o hello hello.cpp `wx-config --cxxflags --libs`
+
+Please note that you *must* use `wx-config` to obtain the compiler and linker
```

Comment:
This comes a bit strong to me, as it may not be clear this still belongs (does it?) to Unix-like systems; in particular considering wx-config is not provided on MSW and generally not used for example with popular Code::Blocks?

Reply:
I actually wanted to be strong (I even hesitated to use `**must**`!) because I keep seeing people trying to manually tweak include paths, header locations etc just because they don't use wx-config correctly and I want to dissuade them from doing it.

But I guess we need to make an exception for IDEs. Would you have any specific advice for C::B by chance? TIA!

Reply:
FWIW, the other popular wxWidgets-based IDE, CodeLite, does use wxConfig even on Windows, it ships with its own version on it.

I am not really familiar with Visual Studio Code, but I have recently looked into using wxConfig with it. I could not find a way how to do that, but that may be on me.

As for Code::Blocks, my advice would be to use its wxWidgets project wizard which sets everything properly based on the user input (I even wrote [a guide](https://github.com/PBfordev/wxpbguide) for that). However, I do understand that this does not cover adding wxWidgets to an existing project and saying "Just copy it from the project created by the wizard" is not acceptable.

My main gripe was insisting on using a tool, which is not shipped (on MSW) with wxWidgets or a compiler toolchain.